### ISSUE TITLE
Upgrade from C128 -> C129 on Win x64 via 1.71.x - Nightly (Checking C129)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#40939]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.71.73 Chromium: 129.0.6668.31 (Official Build) nightly (64-bit)
-- | --
Revision | f6000c6537cef475d001df5d3674ae1fb9d8afd7
OS | Windows 11 Version 23H2 (Build 22631.4112)
```
<details>
<summary>C128 --> C129 on-demand upgrade check - PASSED</summary>

1. installed latest nightly `   ` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `1.71.73 Chromium: 129.0.6668.31 (Official Build) nightly (64-bit) ` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 6`**
----------------|----------------|----------------|---------------|------------


</details>

<details>
<summary>C128 --> C129 background upgrade check - PASSED</summary>

1. installed latest nightly `    ` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
-   * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{B9953B68-A26F-4EB2-BE09-CF9409609028}` via `Task Scheduler`
5. waited ~30 min or so before launching Brave and ensured it was updated to `1.71.73 Chromium: 129.0.6668.31 (Official Build) nightly (64-bit)`  (removed `last checked` in `BraveSoftware\Update\` 
6. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|---------------|----------------|-----------

</details>

<details>
<summary>C129 --> C129 on-demand upgrade check - PASSED</summary>

1. installed `1.71.73 Chromium: 129.0.6668.31 (Official Build) nightly (64-bit)` 
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:

- * `https://updates.brave.com/service/update2`

3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `   ` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|----------------|---------------|----------
 
</details>

<details>
<summary>C129 --> C129 background upgrade check - PASSED</summary>

1. installed `1.71.73 Chromium: 129.0.6668.31 (Official Build) nightly (64-bit) ` and closed Brave
2.  created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
3.  set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{FCFE660D-D369-4717-9908-83F75DF38808}` via `Task Scheduler`
5. closed Brave
6. waited ~30min or so before launching Brave and ensured it was updated to `    `
7. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 7`**  
----------------|----------------|------------|------------|--------- 


</details>

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.71.75 Chromium: 129.0.6668.31 (Official Build) nightly (32-bit)
-- | --
Revision | 3516cf142a88f8a433ef3eb80082fe329e390d01
OS | Windows 10 Version 22H2 (Build 19045.4651)
```

<details>
<summary>C128 --> C129 on-demand upgrade check - PASSED</summary>

* installed `1.71.66 Chromium: 128.0.6613.120` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.71.75 Chromium: 129.0.6668.31` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![image](https://github.com/user-attachments/assets/1f848513-dc9b-4ec9-87bb-9616a37d42f7) | ![image](https://github.com/user-attachments/assets/cc250108-f571-47b1-8afa-1711497961b4) | ![image](https://github.com/user-attachments/assets/56be1d69-93c9-4e45-b777-e33d559332dd) | ![image](https://github.com/user-attachments/assets/07dcaed8-65dc-48cf-ad0b-9e313a4cb603)
</details>

<details>
<summary>C129 --> C129 on-demand upgrade check - PASSED</summary>

* installed `1.71.73 Chromium: 129.0.6668.31` via `BraveBrowserStandaloneNightlySetup32_129_1_71_73.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.71.75 Chromium: 129.0.6668.31` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|----------------
![image](https://github.com/user-attachments/assets/9ee325a3-92ca-4c9d-8eca-4c7fa60f34cf) | ![image](https://github.com/user-attachments/assets/2c32f43f-19a0-468d-97cc-62b9111bb920) | ![image](https://github.com/user-attachments/assets/f9fe47d6-9b1e-433b-a8e5-a0bd5c19dee4) | ![image](https://github.com/user-attachments/assets/c48c884a-5162-4937-9f8f-c7d1f19d1f60)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[09/09/24 23:33:52.754][BraveUpdate:goopdate][3744:8836][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{20A71E8E-DC3C-420B-BDA4-DF04DBF518CF}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[09/09/24 23:33:52.754][BraveUpdate:goopdate][3744:8836][Running installer][C:\Program Files\BraveSoftware\Update\Install\{20A71E8E-DC3C-420B-BDA4-DF04DBF518CF}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[09/09/24 23:33:54.137][BraveUpdate:psmachine][5204:9072][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[09/09/24 23:33:54.138][BraveUpdate:psmachine][5204:9072][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[09/09/24 23:33:54.138][BraveUpdate:psmachine][5204:9072][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[09/09/24 23:33:54.280][BraveUpdate:goopdate][3744:4732][Skipping app, external updater running][{C6CB981E-DB30-4876-8639-109F8933582C}]
[09/09/24 23:33:54.280][BraveUpdate:goopdate][3744:4732][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[09/09/24 23:33:54.280][BraveUpdate:goopdate][3744:4732][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[09/09/24 23:33:54.280][BraveUpdate:goopdate][3744:4732][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[09/09/24 23:33:54.414][BraveUpdate:goopdate][3744:9980][LogEventHelper][4][12][1][[
Update check. Status = 0x00000000.
pid=3744, ver=1.3.361.151, machine=1, extern=1, private
url=https://updates.brave.com/service/update2
App={B131C935-9BE6-41DA-9599-1F776BEB8019}, Ver=1.3.361.151, Status=no-update
]][]
[09/09/24 23:33:54.414][BraveUpdate:goopdate][3744:9980][UpdateLastChecked][now 1725939234]
[09/09/24 23:33:54.496][BraveUpdate:goopdate][5204:9072][DllEntry exit][0x00000000]
[09/09/24 23:34:00.367][BraveUpdate:goopdate][3744:8836][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[09/09/24 23:34:00.367][BraveUpdate:goopdate][3744:8836][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[09/09/24 23:34:00.369][BraveUpdate:goopdate][3744:8836][Started process][9616]
[09/09/24 23:34:00.372][BraveUpdate:goopdate][3744:8836][LogEventHelper][4][12][1][[
Application update/install.
pid=3744, ver=1.3.361.151, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=129.1.71.73, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=129.1.71.73, Install result=0x00000000
]][]
[09/09/24 23:34:00.372][BraveUpdate:goopdate][3744:9980][LogEventHelper][4][12][1][[
Application update/install.
pid=3744, ver=1.3.361.151, machine=1, extern=1, private]][]
[09/09/24 23:34:00.379][BraveCrashHandler][9616:5304][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[09/09/24 23:34:00.380][BraveCrashHandler][9616:5304][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[09/09/24 23:34:00.380][BraveCrashHandler][9616:5304][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[09/09/24 23:34:00.381][BraveCrashHandler][9616:5304][CrashHandler][Instance is already running][32-bit][1]
[09/09/24 23:34:00.540][BraveUpdate:goopdate][8600:6664][DllEntry exit][0x00000000]
[09/09/24 23:34:00.541][BraveUpdate:goopdate][3744:8836][Started process][7984]
[09/09/24 23:34:00.555][BraveUpdate:goopdate][7984:4460][DllEntry]["C:\Program Files\BraveSoftware\Update\BraveUpdate.exe" /ping PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48cmVxdWVzdCBwcm90b2NvbD0iMy4wIiB1cGRhdGVyPSJPbWFoYSIgdXBkYXRlcnZlcnNpb249IjEuMy4zNjEuMTUxIiBzaGVsbF92ZXJzaW9uPSIxLjMuMzYxLjE1MSIgaXNtYWNoaW5lPSIxIiBzZXNzaW9uaWQ9IntDNTJGMEM1Ri05NTNGLTQzMDItQjAzOC0yODFFRTFFMzZCQzl9IiB1c2VyaWQ9Ins5QkM4NjlDOC1CRTdELTQyRDYtQTM4Qy00NjE3N0IzREM1RTV9IiBpbnN0YWxsc291cmNlPSJ1cGRhdGUzd2ViLW9uZGVtYW5kIiB0ZXN0c291cmNlPSJhdXRvIiByZXF1ZXN0aWQ9IntGQzBEMzY0MS1CRDA3LTRCQUQtQkY0Ny03OUFFNjE3MUZBQzd9IiBkZWR1cD0iY3IiIGRvbWFpbmpvaW5lZD0iMCI-PGh3IHBoeXNtZW1vcnk9IjMiIHNzZT0iMSIgc3NlMj0iMSIgc3NlMz0iMSIgc3NzZTM9IjEiIHNzZTQxPSIxIiBzc2U0Mj0iMSIgYXZ4PSIxIi8-PG9zIHBsYXRmb3JtPSJ3aW4iIHZlcnNpb249IjEwLjAuMTkwNDUuNDY1MSIgc3A9IiIgYXJjaD0ieDg2Ii8-PGFwcCBhcHBpZD0ie0M2Q0I5ODFFLURCMzAtNDg3Ni04NjM5LTEwOUY4OTMzNTgyQ30iIHZlcnNpb249IjEyOS4xLjcxLjczIiBuZXh0dmVyc2lvbj0iMTI5LjEuNzEuNzUiIGFwPSJuaWdodGx5IiBsYW5nPSIiIGJyYW5kPSJHR0xTIiBjbGllbnQ9IiIgaW5zdGFsbGFnZT0iMCIgaW5zdGFsbGRhdGU9IjY0NjEiPjxldmVudCBldmVudHR5cGU9IjEyIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTMiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiLz48ZXZlbnQgZXZlbnR0eXBlPSIxNCIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIgZG93bmxvYWRlcj0iYml0cyIgdXJsPSJodHRwczovL3VwZGF0ZXMtY2RuLmJyYXZlLmNvbS9kZWx0YS9CcmF2ZS1Ccm93c2VyLU5pZ2h0bHkvbmlnaHRseS93aW4vMTI5LjEuNzEuNzUvMTI5LjEuNzEuNzMveDg2L2JyYXZlX2luc3RhbGxlci1kZWx0YS1pYTMyLmV4ZSIgZG93bmxvYWRlZD0iMzYzMjE1MiIgdG90YWw9IjM2MzIxNTIiIGRvd25sb2FkX3RpbWVfbXM9IjEzOTYiLz48ZXZlbnQgZXZlbnR0eXBlPSIxNCIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIvPjxldmVudCBldmVudHR5cGU9IjE1IiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMyIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMTk2NjA5IiBzb3VyY2VfdXJsX2luZGV4PSIwIiB1cGRhdGVfY2hlY2tfdGltZV9tcz0iMzcwIiBkb3dubG9hZF90aW1lX21zPSIxNDcyIiBkb3dubG9hZGVkPSIzNjMyMTUyIiB0b3RhbD0iMzYzMjE1MiIgaW5zdGFsbF90aW1lX21zPSI3NjEzIi8-PC9hcHA-PC9yZXF1ZXN0Pg]
[09/09/24 23:34:00.556][BraveUpdate:goopdate][7984:4460][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[09/09/24 23:34:00.556][BraveUpdate:goopdate][7984:4460][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[09/09/24 23:34:00.556][BraveUpdate:goopdate][7984:4460][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[09/09/24 23:34:00.557][BraveUpdate:goopdate][7984:4460][C:\Program Files\BraveSoftware\Update\1.3.361.151\goopdate.dll][version 1.3.361.151][opt][dev]
[09/09/24 23:34:00.558][BraveUpdate:goopdate][7984:4460][is machine: 1]
[09/09/24 23:34:01.015][BraveUpdate:goopdate][7984:4460][DllEntry exit][0x00000000]
[09/09/24 23:34:01.020][BraveUpdate:goopdate][3744:3472][DllEntry exit][0x00000000]
```
</details>

<details>
<summary>C128 --> C129 background upgrade check - PASSED</summary>

* installed `1.71.66 Chromium: 128.0.6613.120` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{C652585D-72D4-47C2-ADD7-E4154C822BD1}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.71.75 Chromium: 129.0.6668.31`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|-----------------
![image](https://github.com/user-attachments/assets/a12997e3-330a-4bc7-8b3e-0015522e871e) | ![image](https://github.com/user-attachments/assets/9533247d-20b1-4184-8987-20d25ca54707) | ![image](https://github.com/user-attachments/assets/8dfbe4c9-acb2-4bb6-94b9-e31c63ca072d) | ![image](https://github.com/user-attachments/assets/40c8bbdd-8b41-432e-bb70-82e8d9e7f11e)
</details>

<details>
<summary>C129 --> C129 background upgrade check - PASSED</summary>

* installed `1.71.73 Chromium: 129.0.6668.31` via `BraveBrowserStandaloneNightlySetup32_129_1_71_73.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{255700BD-7CBB-476F-95DB-05324065749A}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.71.75 Chromium: 129.0.6668.31`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/user-attachments/assets/2bd842a7-e340-44f1-a68a-29794f7d916f) | ![image](https://github.com/user-attachments/assets/37f864ea-f808-48ed-80ee-eef383d4ae09) | ![image](https://github.com/user-attachments/assets/62642e9f-935e-430c-9d40-2630c0123e3e) | ![image](https://github.com/user-attachments/assets/9c96958f-0a89-47d5-a1b4-1d85b4e41392)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[09/09/24 23:53:13.706][BraveUpdate:goopdate][10936:10176][UpdateLastChecked][now 1725940393]
[09/09/24 23:53:31.315][BraveUpdate:goopdate][10936:10176][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{7A397394-A4A1-4577-8A78-30D1A5AC3714}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 0]
[09/09/24 23:53:31.315][BraveUpdate:goopdate][10936:10176][Running installer][C:\Program Files\BraveSoftware\Update\Install\{7A397394-A4A1-4577-8A78-30D1A5AC3714}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[09/09/24 23:53:48.878][BraveUpdate:goopdate][10936:10176][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[09/09/24 23:53:48.878][BraveUpdate:goopdate][10936:10176][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 2][]["C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe" --from-installer]
[09/09/24 23:53:48.940][BraveUpdate:goopdate][10936:10176][Started process][4388]
[09/09/24 23:53:48.940][BraveUpdate:goopdate][10936:10176][LogEventHelper][4][12][1][[
Application update/install.
pid=10936, ver=1.3.361.151, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=129.1.71.73, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=129.1.71.73, Install result=0x00000000
]][]
[09/09/24 23:53:48.940][BraveCrashHandler][4388:9264][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[09/09/24 23:53:48.940][BraveCrashHandler][4388:9264][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[09/09/24 23:53:48.940][BraveCrashHandler][4388:9264][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[09/09/24 23:53:48.956][BraveCrashHandler][4388:9264][CrashHandler::RunUntilShutdown]
[09/09/24 23:53:48.987][BraveUpdate:goopdate][9804:9420][DllEntry exit][0x00000000]
[09/09/24 23:53:49.003][BraveUpdate:goopdate][10936:5892][Started process][9552]
[09/09/24 23:53:49.019][BraveUpdate:goopdate][9552:7220][DllEntry]["C:\Program Files\BraveSoftware\Update\BraveUpdate.exe" /ping PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48cmVxdWVzdCBwcm90b2NvbD0iMy4wIiB1cGRhdGVyPSJPbWFoYSIgdXBkYXRlcnZlcnNpb249IjEuMy4zNjEuMTUxIiBzaGVsbF92ZXJzaW9uPSIxLjMuMzYxLjE1MSIgaXNtYWNoaW5lPSIxIiBzZXNzaW9uaWQ9Ins3RDk0RjZDNC0wMDJDLTREODAtODcwNi01MzQxNjZBOUVEMjZ9IiB1c2VyaWQ9Ins4OEMzNzRFQS1BMDFELTQ0RTAtQUZCQi1GNEIwOUJFNDk5NUJ9IiBpbnN0YWxsc291cmNlPSJzY2hlZHVsZXIiIHRlc3Rzb3VyY2U9ImF1dG8iIHJlcXVlc3RpZD0ie0FDNDc5NEM5LUIzNDAtNEZDNC1BM0RDLUI4Q0EyNkEwQkIwRn0iIGRlZHVwPSJjciIgZG9tYWluam9pbmVkPSIwIj48aHcgcGh5c21lbW9yeT0iMyIgc3NlPSIxIiBzc2UyPSIxIiBzc2UzPSIxIiBzc3NlMz0iMSIgc3NlNDE9IjEiIHNzZTQyPSIxIiBhdng9IjEiLz48b3MgcGxhdGZvcm09IndpbiIgdmVyc2lvbj0iMTAuMC4xOTA0NS40NjUxIiBzcD0iIiBhcmNoPSJ4ODYiLz48YXBwIGFwcGlkPSJ7QzZDQjk4MUUtREIzMC00ODc2LTg2MzktMTA5Rjg5MzM1ODJDfSIgdmVyc2lvbj0iMTI5LjEuNzEuNzMiIG5leHR2ZXJzaW9uPSIxMjkuMS43MS43NSIgYXA9Im5pZ2h0bHkiIGxhbmc9IiIgYnJhbmQ9IkdHTFMiIGNsaWVudD0iIiBpbnN0YWxsYWdlPSIwIj48ZXZlbnQgZXZlbnR0eXBlPSIxMiIgZXZlbnRyZXN1bHQ9IjEiIGVycm9yY29kZT0iMCIgZXh0cmFjb2RlMT0iMCIvPjxldmVudCBldmVudHR5cGU9IjEzIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTQiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiIGRvd25sb2FkZXI9ImJpdHMiIHVybD0iaHR0cHM6Ly91cGRhdGVzLWNkbi5icmF2ZS5jb20vZGVsdGEvQnJhdmUtQnJvd3Nlci1OaWdodGx5L25pZ2h0bHkvd2luLzEyOS4xLjcxLjc1LzEyOS4xLjcxLjczL3g4Ni9icmF2ZV9pbnN0YWxsZXItZGVsdGEtaWEzMi5leGUiIGRvd25sb2FkZWQ9IjM2MzIxNTIiIHRvdGFsPSIzNjMyMTUyIiBkb3dubG9hZF90aW1lX21zPSIxNjk4NCIvPjxldmVudCBldmVudHR5cGU9IjE0IiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIwIi8-PGV2ZW50IGV2ZW50dHlwZT0iMTUiIGV2ZW50cmVzdWx0PSIxIiBlcnJvcmNvZGU9IjAiIGV4dHJhY29kZTE9IjAiLz48ZXZlbnQgZXZlbnR0eXBlPSIzIiBldmVudHJlc3VsdD0iMSIgZXJyb3Jjb2RlPSIwIiBleHRyYWNvZGUxPSIxOTY2MDkiIHNvdXJjZV91cmxfaW5kZXg9IjAiIHVwZGF0ZV9jaGVja190aW1lX21zPSIyNTQ3IiBkb3dubG9hZF90aW1lX21zPSIxNzA3OCIgZG93bmxvYWRlZD0iMzYzMjE1MiIgdG90YWw9IjM2MzIxNTIiIGluc3RhbGxfdGltZV9tcz0iMTc1NjMiLz48L2FwcD48L3JlcXVlc3Q-]
[09/09/24 23:53:49.019][BraveUpdate:goopdate][9552:7220][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[09/09/24 23:53:49.019][BraveUpdate:goopdate][9552:7220][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[09/09/24 23:53:49.019][BraveUpdate:goopdate][9552:7220][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[09/09/24 23:53:49.019][BraveUpdate:goopdate][9552:7220][C:\Program Files\BraveSoftware\Update\1.3.361.151\goopdate.dll][version 1.3.361.151][opt][dev]
[09/09/24 23:53:49.019][BraveUpdate:goopdate][9552:7220][is machine: 1]
[09/09/24 23:53:49.550][BraveUpdate:goopdate][9552:7220][DllEntry exit][0x00000000]
[09/09/24 23:53:49.550][BraveUpdate:goopdate][10936:3600][DllEntry exit][0x00000000]
```
</details>
